### PR TITLE
EPMDJ-7969: Add dockerfile and nginx.config

### PR DESCRIPTION
**Pull request by IliaSolovev**
Update package name because the previous package was delete

### CODE REVIEW

File: docker-compose.services.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      - LOG_LEVEL=trace
+      - DRILL_DEFAULT_PACKAGES=org/springframework/samples/petclinic
+    ports:
+      - 9090:8090
+      - 6006:5006
+    networks:
+      - drill4j-dev-network
+
+  admin-ui:
+    image: drill4j/admin-ui
```
Comment (IliaSolovev) [inline docker-compose.services.yml:18]: This image does not exist now, because i not sure that it will have so name

Comment (IliaSolovev) [reply docker-compose.services.yml:18]: What your suggestion?

File: .dockerignore [hunk 2]
```diff
@@ -0,0 +1,9 @@
+**/node_modules
+**/coverage
+**/common-hooks
+**/common-utils
+**/send-notification-event
+
+**/npm-debug.log*
+**/yarn-debug.log*
```
Comment (RomanDavlyatshin) [inline .dockerignore:8]: yarn is not utilized in this project. I suggest removing that.

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+ENV REACT_APP_API_HOST "$API_HOST"
+WORKDIR /app
+COPY . /app
+RUN npm install --silent
+RUN npm run build
+WORKDIR /app/ui
+RUN npm install --silent
+RUN npm run build
+WORKDIR /app
+RUN mv ui/dist dist/ui
```
Comment (RomanDavlyatshin) [inline Dockerfile:17]: I don't see a point in adding `ui` folder here

File: docker-compose.services.yml (hunk 1 above)
Comment (RomanDavlyatshin) [reply docker-compose.services.yml:18]: 1. We can publish any image at any moment and just delete it later
2. `admin-ui-container` ?

File: src/index.ejs [hunk 4]
```diff
@@ -46,8 +46,19 @@
...
+        "react-dom": "https://cdn.jsdelivr.net/npm/react-dom@17/umd/react-dom.development.min.js"
+      }
+    }
+  </script>
+  <% } else { %>
+    <script type="systemjs-importmap">
+    {
+      "imports": {
+        "@drill4j/admin-ui-container": "/Drill4J-admin-ui-container.js",
+        "@drill4j/admin-ui": "/ui/Drill4J-admin-ui.js",
```
Comment (RomanDavlyatshin) [inline src/index.ejs:61]: I suggest removing `/ui/` prefix

File: src/index.ejs [hunk 5]
```diff
@@ -46,8 +46,19 @@
   <script type="systemjs-importmap">
     {
       "imports": {
-        "@drill4j/root-config": "//localhost:9000/Drill4J-root-config.js",
-        "@drill4j/admin-ui": "http://localhost:8080/Drill4J-admin-ui.js",
+        "@drill4j/admin-ui-container": "//localhost:9000/Drill4J-admin-ui-container.js",
```
Comment (RomanDavlyatshin) [inline src/index.ejs:49]: These magic port numbers trouble me, we should discuss that

File: ui/src/index.css [hunk 6]
```diff
@@ -1,4 +1,4 @@
-/*! tailwindcss v2.2.2 | MIT License | https://tailwindcss.com */
+/*! tailwindcss v2.2.4 | MIT License | https://tailwindcss.com */
```
Comment (RomanDavlyatshin) [inline ui/src/index.css:1]: Why are we adding the generated tailwind css again? 😢 
It is created at build-time, I don't see a reason to commit it.

**Review by RomanDavlyatshin — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 3 above)
Comment (IliaSolovev) [reply Dockerfile:17]: It will remove in the next pr, when i update building apps

**Pull request merged**